### ISSUE TITLE
Non-list JSONata output causes error

### ISSUE THREAD

**Issue by ASL-rmarshall** [cdisc-org/cdisc-rules-engine#1395]
At the moment, an error is raised if the processed JSONata statements do not return a list: https://github.com/cdisc-org/cdisc-rules-engine/blob/14828f85c535ef1bc0b9d5155aff503a0d1b12fd/cdisc_rules_engine/utilities/jsonata_processor.py#L50-L54

This causes problems because, by default, JSONata converts an array with a single item into a non-array.  To prevent this error, it would be necessary to explicitly tell JSONata to return an array/list for every rule that might return only a single issue (e.g., by adding `[]` to the end of the output clause).  It would be better if non-array/list results are just converted to a list for subsequent processing:
```python
            if not isinstance(results, list):
                results = [results]
```

**Comment by ASL-rmarshall** [comment]
@gerrycampion Can this be implemented as a quick fix?
